src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -366,26 +366,78 @@ function showNewlyAddedResponseCommentEditForm(addedIndex) {
     $("#responseCommentEditForm-" + addedIndex).show();
 }
 
-function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
+function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, clickedElement) {
     $(".tooltip").hide();
-    var panelBody = $(sender).parent().find('div[class^="panel-body"]');
+    var $clickedElement = $(clickedElement);
+    var $collapsiblePanel = $clickedElement.siblings(".collapse");
+    var panelBody = $clickedElement.parent().find('div[class^="panel-body"]');
     var fsNameForUrl = encodeURIComponent(fsName);
     var url = "/page/instructorFeedbackResponseCommentsLoad?user=" + user + "&courseid=" + courseId + "&fsname=" + fsNameForUrl + "&fsindex=" + fsIndx;
-    $(sender).find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
+    
+    // If the content is already loaded, toggle the chevron and exit.
+    if ($clickedElement.hasClass("loaded")) {
+        clearQueueAndToggleCollapsiblePanel($collapsiblePanel);
+        toggleChevron(clickedElement);
+        
+        return;
+    }
+    
+    $clickedElement.find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
+    
     panelBody.load(url, function( response, status, xhr ) {
-        if (status == "success") {
-            panelBody.removeClass('hidden');
+        if (status !== "success") {
+            panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
+        } else {
             updateBadgeForPendingComments(panelBody.children(":first").text());
             panelBody.children(":first").remove();
             registerResponseCommentsEvent();
             registerCheckboxEventForVisibilityOptions();
             enableHoverToDisplayEditOptions();
-      } else {
-          panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
-          panelBody.removeClass('hidden');
-      }
-      $(sender).find('div[class^="placeholder-img-loading"]').html("");
+
+            $clickedElement.addClass("loaded");
+        }
+
+        clearQueueAndToggleCollapsiblePanel($collapsiblePanel);
+        $clickedElement.find('div[class^="placeholder-img-loading"]').html("");
+        toggleChevron(clickedElement);
     });
+    
+}
+
+// Clears the animation queue of the panel before collapsing / expanding the panel.
+function clearQueueAndToggleCollapsiblePanel(collapsiblePanel) {
```

Comment:
I don't think you need to mention 'clearQueue' in the function name. It's part of the 'ToggleCollapsiblePanel' action, right? Furthermore, you didn't mention 'clearQueue' in the function name of 'toggleChevron' below.